### PR TITLE
Fixing task versions jump for ARG task

### PR DESCRIPTION
**Pull request by issacnitin**

### CODE REVIEW

File: Tasks/AzureResourceGroupDeploymentV2/task.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 2,
         "Minor": 157,
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeploymentV2/task.json]: change it to 161

File: Tasks/AzureResourceGroupDeploymentV2/task.loc.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   "version": {
     "Major": 2,
     "Minor": 157,
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeploymentV2/task.loc.json]: this also

File: Tasks/AzureResourceGroupDeploymentV2/task.json (hunk 1 above)
Comment (issacnitin) [reply Tasks/AzureResourceGroupDeploymentV2/task.json]: Done

File: Tasks/AzureResourceGroupDeploymentV2/task.json [hunk 3]
```diff
@@ -14,8 +14,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 157,
-        "Patch": 14
+        "Minor": 161,
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeploymentV2/task.json]: change version to 162.0
also update releases/m162 branch to vrsion 162.0

Comment (bishal-pdMSFT) [reply Tasks/AzureResourceGroupDeploymentV2/task.json]: Sorry, this comment is not applicable for M161 PR

**Pull request merged**